lib/req-res/status.js

```diff
@@ -8,23 +8,16 @@ module.exports = function ( options ) {
 	return {
 		'success' : function ( data ) {
 
-			log.info( 'success', data );
-
 			respond( {
 				'status' : 'success',
 				'data'   : data
 			} );
+
+			log.info( 'success', data );
```

Comment:
Do we want this as `info`? I think our default level would be `info`. I'd think that this would only be at `debug` or `silly`.